tuf/api/metadata.py

```diff
@@ -415,6 +441,50 @@ def remove_key(self, role: str, keyid: str) -> None:
             del self.keys[keyid]
 
 
+class MetadataInfo:
+    """A container with information about a particular metadata file.
+    Instances of MetadataInfo are used as values in a dictionary called
+    "meta" in Timestamp and Snapshot.
+
+    Attributes:
+        version: An integer indicating the version of the metadata file.
+        length: An optional integer indicating the length of the metadata file.
+        hashes: A optional dictionary containing hash algorithms and the
+            hashes resulting from applying them over the metadata file.::
```

Comment:
I wonder if something like the following might be clearer? Or am I nitpicking?

```suggestion
        hashes: An optional dictionary mapping hash algorithms to the
            hashes resulting from applying them over the metadata file
            contents.::
```